src/participant/markdown.ts

```diff
@@ -6,6 +6,8 @@ export function createMarkdownLink({
   name,
 }: {
   commandId: string;
+  // TODO: Create types for this data so we can also then use them on the extension
+  // controller when we parse the result.
```

Comment:
Maybe we could also update this method by allowing to create not only command link like `"command:..."`, but also regular links like `"https://..."`.

Reply:
Is that for the docs command?

Reply:
For example yes. I can add this in my pr as well, just wanted to discuss it with you first, if this makes sense to you.